test-it/ExpectedResults/Waveform.vtr

```diff
@@ -1,35 +1,35 @@
 Analog Waveform Testing
 ***********************
-Empty analog waveform a_wave = ((0 0) 0 ())
+Empty analog waveform a_wave = ((0 0) 0 () ^Variant)
 Analog parts (0 0) 0 ()
-Analog waveform a_wave = ((0 0) 0 (1.2 1.3 1 -0.5))
-Analog waveform a_wave = ((3566073600 123) 5.8 (1.2 1.3 1 -0.5))
+Analog waveform a_wave = ((0 0) 0 (1.2 1.3 1 -0.5) ^Variant)
```

Comment:
This works now, but the `^Variant` doesn't read well. I think it is because variants do not have a custom print function?

Reply:
Fixed this to allow Variants be printed in the following format:
`{"value": <expand-underlying-type>, "attributes":["foo": <expand-attribute-value>, "bar": <expand attribute value>]}`